atomicfu/src/appleMain/kotlin/kotlinx/atomicfu/locks/NativeMutexNode.kt

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2024 The Android Open Source Project
```

Comment:
That's not the correct copyright, please use one from the repo

Reply:
(the same comment applies for the original lock implementation)

Reply:
Done